policyengine_uk/parameters/gov/hmrc/income_tax/adjusted_net_income_components.yaml

```diff
@@ -0,0 +1,19 @@
+description: Components of adjusted net income
+values:
+  0000-01-01:
+    - taxable_employment_income
+    - taxable_pension_income
+    - taxable_social_security_income
+    - taxable_self_employment_income
+    - taxable_property_income
+    - taxable_savings_interest_income
+    - taxable_dividend_income
+    - taxable_miscellaneous_income
+    
+metadata:
+  unit: program
```

Comment:
```suggestion
  unit: list
```

Reply:
generally we try to consolidate those parameter files into 
unit: list 

We used a bunch of versions over the years but this is what we settled on

Reply:
all yaml